app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/ThrottlingRate.tsx

```diff
@@ -0,0 +1,205 @@
+"use client";
+import { useEffect, useState } from "react";
+import { useTranslation } from "@i18n/client";
+import { Alert, Button } from "@clientComponents/globals";
+import {
+  getThrottling,
+  permanentThrottling,
+  resetThrottling,
+  scheduledThrottling,
+} from "@lib/cache/throttlingCache";
+import { Checkbox } from "@formBuilder/components/shared/MultipleChoice";
+import { Input } from "@formBuilder/components/shared/Input";
+import { toast, ToastContainer } from "@formBuilder/components/shared/Toast";
+import { getSecondsInWeeks, getWeeksInSeconds } from "@lib/utils/date/dateConversions";
+import { SubmitButton } from "@clientComponents/globals/Buttons/SubmitButton";
+import { Tooltip } from "@formBuilder/components/shared/Tooltip";
+import { useRehydrate } from "@lib/store/useTemplateStore";
+import Skeleton from "react-loading-skeleton";
+import { formClosingDateEst } from "@lib/utils/date/utcToEst";
+import { logMessage } from "@lib/logger";
+
+export const ThrottlingRate = ({ formId }: { formId: string }) => {
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation("admin-settings");
+  const hasHydrated = useRehydrate();
+  const [hasPreLoaded, setHasPreLoaded] = useState(false);
```

Comment:
Can we change this to 

```
const [isLoading, setIsLoading] = useState(false);
```

Preloading makes me think we're loading ahead of time

Reply:
This is used for loading the setting. It kind of is loading data ahead of time. What about `loadingSetting`?